README.md

```diff
@@ -0,0 +1,14 @@
+# COMIT research outcomes
+
+List of spike outcomes for COMIT network.
+
+- [spike-0000](0000-use-madr-like-solution-for-spike-outcomes.md) - Use MADR-like solution for spike outcomes - coblox/docs#36
+- [spike-0001](0001-basic-expiry-model.md) - Propose basic model to calculate expiry times - comit-network/RFCs#14
+- [spike-0002](0002-ether-htlc-dynamic-final-addresses.md) - Ether HTLC with dynamic final addresses
+- [spike-0003](0003-lightning-as-alpha-ledger.adoc) - LND as Alpha Ledger as described in RFC003
+- [spike-0004](0004-siren-prototype-accept-decline.adoc) - Siren Prototype for comit-rs HTTP API - Accept/Decline actions
+- [spike-0005](0005-kzen-two-party-ecdsa.adoc) - Kzen Two Party ECDSA Review
+- [spike-0006](0006-scriptless-scripts-with-ecdsa.adoc) - Scriptless Scripts with ECDSA
+- [spike-0008](0007-secret-seed-storage-and-key-derivation.md) - Secret seed storage and key derivation
```

Comment:
```suggestion
- [spike-0007](0007-secret-seed-storage-and-key-derivation.md) - Secret seed storage and key derivation
```
Note: it seems spike `007` is actually not merged yet, see: https://github.com/comit-network/spikes/ - should this line be removed for now?

Reply:
It is merged but as a number clash with `0003`.

Reply:
Fixed, thanks.